doc/strength-and-numbers.md

```diff
@@ -17,6 +23,20 @@ I'm using "E" instead of "×10^" notation.  The number after the E is the number
 
 * 4E78 - 4E82: Estimate for the number of atoms in the observable universe.
 
+
+Random Tokens
+-------------
+
+Account IDs, tokens and other randomly generated things on the system generate a secure hash of a particular length.  By default we use 24 random bytes of information, which translates into 32 bytes of Base64 encoded data.  Trying to guess a valid account is very hard.  In 24 bytes there are 192 bits (24 * 8 = 192).  All of them have random values (0 or 1).  That means there are 2^192 possible acounts.  Roughly 6.2E57 accounts are possible.  That's more than one account for each atom on a million Earths.
```

Comment:
Should it say "... to guess a valid account ID/id is very hard."?

Misspelled "acounts" after "... there are 2^192 possible..."